custom/abt/reports/late_pmt.py

```diff
@@ -194,9 +194,6 @@ def _to_report_format(date, user, group):
                 group
             ]
 
-        def not_in_group(key, group):
-            return key not in group
```

Comment:
:sob:

Reply:
the resulting `not not_in_group(key, group_a)` below was especially :eyes: